### PR TITLE
More TSK previews

### PR DESCRIPTION
**Pull request by zzorba**

### CODE REVIEW

File: pack/tsk/tskp.json [hunk 1]
```diff
@@ -413,6 +413,40 @@
...
+        "illustrator": "Ethan Patrick Harris",
+        "name": "Orphic Theory",
+        "pack_code": "tskp",
+        "position": 51,
+        "quantity": 2,
+        "skill_intellect": 1,
+        "slot": "Arcane",
+        "text": "Uses (4 secrets).\n[fast] Spend 1 secret: Choose any non-weakness treachery not attached to an [[Elite]] enemy. Until the end of the round, treat that card's printed text box as if it were blank (except for [[Traits]]).",
+        "traits": "Spell.",
+        "type_code": "event",
```
Comment (drawntotheflame) [inline pack/tsk/tskp.json:447]:
```suggestion
        "type_code": "asset",
```

**Review by drawntotheflame — COMMENTED** [review]
Good job, one change.

File: pack/tsk/tskp.json [hunk 2]
```diff
@@ -395,6 +395,27 @@
...
         "xp": 2
     },
+    {
+        "code": "09038",
+        "cost": 4,
+        "deck_limit": 2,
+        "faction_code": "guardian",
+        "health": 3,
+        "illustrator": "Tiziano Baracchi",
+        "name": "Girish Kadakia",
```
Comment (drawntotheflame) [inline pack/tsk/tskp.json:405]: Add:
`"is_unique": true,`

File: pack/tsk/tskp.json [hunk 3]
```diff
@@ -584,6 +657,24 @@
         "type_code": "asset",
         "xp": 0
     },
+    {
+        "code": "09086",
+        "cost": 1,
+        "deck_limit": 2,
+        "faction_code": "mystic",
+        "flavor": "By both light and shadow / may dusk beget dawn; May the turning of the world / lead to power everlasting.",
```
Comment (drawntotheflame) [inline pack/tsk/tskp.json:665]: Should we add line breaks to this, to represent the formatting of the poem? 
```suggestion
        "flavor": "By both light and shadow /\nmay dusk beget dawn;\nMay the turning of the world /\nlead to power everlasting.",
```

Maybe too ott?

Comment (drawntotheflame) [reply pack/tsk/tskp.json:665]: We wouldn't usually, but I wondered because it's formatted like a poem...

File: pack/tsk/tskp.json [hunk 4]
```diff
@@ -670,6 +781,39 @@
...
+        "xp": 0
+    },
+    {
+        "code": "09106",
+        "cost": null,
+        "deck_limit": 2,
+        "faction_code": "survivor",
+        "flavor": "There is a certain clarity in the grim mathematics of survival.",
+        "illustrator": "Lin Hsiang",
+        "name": "Making Preparation",
```
Comment (drawntotheflame) [inline pack/tsk/tskp.json:808]:
```suggestion
        "name": "Making Preparations",
```

File: pack/tsk/tskp.json [hunk 5]
```diff
@@ -685,6 +829,51 @@
...
+    },
+    {
+        "code": "09115",
+        "cost": null,
+        "deck_limit": 2,
+        "faction_code": "survivor",
+        "illustrator": "Quintin Gleim",
+        "name": "Heed the Dream",
+        "pack_code": "tskp",
+        "position": 116,
```
Comment (drawntotheflame) [inline pack/tsk/tskp.json:870]:
```suggestion
        "position": 115,
```

**Review by drawntotheflame — COMMENTED** [review]
Great job @zzorba

**Comment by zzorba** [comment]
All comments addressed.

**Comment by drawntotheflame** [comment]
@Kamalisk this one can go live - thank you.

**Pull request merged**